sway/desktop/fx_renderer.c

```diff
@@ -3,6 +3,9 @@
 
 // TODO: add push / pop_gles2_debug(renderer)?
 
+#include <stdint.h>
+#include <wlr/util/region.h>
+#include "pixman.h"
 #define _POSIX_C_SOURCE 200809L
```

Comment:
Why do we have this? Compiles on my machine without it, and raises an error as we define a different _POSIX_C_SOURCE elsewhere

Reply:
Must've been an auto import when using clang-lsp completion